graphql/resolvers.go

```diff
@@ -972,28 +1005,25 @@ func (r *queryResolver) Projects(ctx context.Context) (*Projects, error) {
 	}
 
 	usr := MustHaveUser(ctx)
-	groupsMap := make(map[string][]*restModel.UIProjectFields)
-	favorites := []*restModel.UIProjectFields{}
+	groupsMap := make(map[string][]*restModel.APIProjectRef)
+	favorites := []*restModel.APIProjectRef{}
 
 	for _, p := range allProjs {
 		groupName := strings.Join([]string{p.Owner, p.Repo}, "/")
-
-		uiProj := restModel.UIProjectFields{
-			DisplayName: p.DisplayName,
-			Identifier:  p.Identifier,
-			Repo:        p.Repo,
-			Owner:       p.Owner,
+		apiProjectRef := restModel.APIProjectRef{}
+		pErr := apiProjectRef.BuildFromService(p)
+		if pErr != nil {
+			return nil, InternalServerError.Send(ctx, fmt.Sprintf("error building APIProjectRef from service: %s", err.Error()))
```

Comment:
This should use pErr
As an aside, I generally recommend declaring these kinds of errors on one line (like `if pErr := apiProjectRef.BuildFromService(p); pRef != nil`), although I see that isn't necessarily the standard in the graphql package